targets/kodi

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+REQUIRES='x11'
+DESCRIPTION='Installs the KODI media player. (Approx. 140MB)'
+HOSTBIN='startkodi'
+CHROOTETC='kodi-keyboard.xml kodi-cycle.py'
+. "${TARGETSDIR:="$PWD"}/common"
+
+### Append to prepare.sh:
+if [ "$DISTRO" = 'debian' ]; then
+    echo "deb http://http.debian.net/debian jessie-backports main" | sudo tee /etc/apt/sources.list.d/kodi.list
+fi
+
+if [ "$DISTRO" = 'ubuntu' ]; then
+    sudo apt-get install -y software-properties-common
+    sudo add-apt-repository -y ppa:team-xbmc/ppa
+fi
+
+if [ "$DISTRO" = 'kali' ]; then
+    error 99 "Kodi not supported on Kali Linux"
+fi
+
+sudo apt-get update
```

Comment:
This should only be run on distros that add the repository (e.g., put it in the if statements above).  Also, it seems apt-get update should be `|| true`'d in case the user has broken repos.